foo_067_rw0_crop_suitability_class/foo_067_rw0_crop_suitability_class_processing.py

```diff
@@ -97,16 +108,20 @@
     d = urllib.request.urlretrieve(url, filename)
     raw_data_file.append(d[0])
 
+# add rice ensembles to list of raw data files
+for rr in raw_rice_ensemble_file:
+    raw_data_file.append(rr)
+
 '''
 Process data
 '''
 # generate names for processed tif files
 processed_data_file = [x[:-4]+'_edit'+x[-4:] for x in raw_data_file]
 
-# rename the tif file
+# rename the tif file and process with gdal
 for raw, processed in zip(raw_data_file, processed_data_file):
-    # set nodata to 255
-    cmd = 'gdalwarp -dstnodata 255 {} {}'.format(raw, processed)
+    # set nodata to -9 (as defined by source in tif)
+    cmd = 'gdalwarp -dstnodata -9 {} {}'.format(raw, processed)
```

Comment:
I think this was a relic from the previous version of the script (with the old tifs). This line was set to `255` for uploading to GEE, so I changed it to `-9` since these new tifs have values that range from -1 to 10,000